addons/web/static/tests/mobile/views/list_view_tests.js

```diff
@@ -85,9 +85,9 @@ QUnit.module("Mobile Views", ({ beforeEach }) => {
         assert.containsOnce(fixture, "div.o_control_panel .o_cp_action_menus");
 
         await toggleActionMenu(fixture);
-        assert.deepEqual(
-            getMenuItemTexts(fixture.querySelector(".o_cp_action_menus")),
-            ["Duplicate", "Delete"],
+        const menuItemTexts = getMenuItemTexts(fixture.querySelector(".o_cp_action_menus"));
+        assert.ok(
+            ["Duplicate", "Delete"].every((menuitem) => menuItemTexts.includes(menuitem)),
```

Comment:
Why do you need to edit tests, without any change in web/ ??
If you do a patch in enterprise, you should apply it only in your test suite.

Reply:
following her patch ,the number of default 'action items' will change based on if the module is installed or not. We patch a very generic menu in Enterprise, the strict equality of the test does not account for such a change. Do you have a suggestion?

Reply:
It s only a js patch ? (not in xml ?) If it s just JS, you can apply the patch only where you need it

Reply:
But not in web/tests/, you want to do something like in /community/addons/hr/static/tests/disable_patch.js. To unpatch it in the test and apply it only in your test suite :)